src/GeositeFramework/locales/en.json

```diff
@@ -15,7 +15,7 @@
     "Force maps to show the same extent": "Force maps to show the same extent",
     "Link Maps": "Link Maps",
     "Receive a printable copy of the current map": "Receive a printable copy of the current map",
-    "Export Page": "Export Page",
+    "Create Map": "Create Map",
```

Comment:
I'm guessing it's this one? -->
https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/prototype/src/GeositeFramework/locales/es.json#L45